astropy/io/votable/tests/data/regression.bin.tabledata.truth.1.1.xml

```diff
@@ -117,7 +117,7 @@
       <TD>4096</TD>
       <TD>268435456</TD>
       <TD>922337203685477</TD>
-      <TD>8.9999990000000007</TD>
+      <TD>8.9990234375</TD>
```

Comment:
I changed the example such that it is `9 - 1/2**10` and thus exactly representable.

Reply:
I think "8.9999990000000007" was designed for a purpose (perhaps to test floating point precision), so I am not comfortable with this change. Is there no way to make test pass without changing the file?

Reply:
I think this is actually OK:
```
In [1]: 8.9999990000000007 == 8.999999
Out[1]: True
```
Previously, numpy just put out a large number of digits, so that one was sure that a number was well-represented, but it didn't not check it could be done with fewer; now it puts out just the right number (see above).

It also seems clear the "truth" was generated once, and is now tested against. The actual *input* file (`regression.xml`) does in fact had `8.999999` (and with numpy 1.14, that is what you get out; older versions are the ones that are problematic).

Reply:
Alright, you're the numpy expert here, so I'll just go with the flow.